src/app/components/school-menu/school-menu.tsx

```diff
@@ -7,41 +7,64 @@ import './school-menu.scss';
 const schoolMenuStaticLinks = [
   {
     title: 'About RS School',
-    detailsUrl: '/courses',
+    detailsUrl: '/#about',
     description: 'Free online education',
   },
   {
     title: 'Upcoming courses',
-    detailsUrl: '/courses#upcoming-courses',
+    detailsUrl: '/#upcoming-courses',
     description: 'Schedule your study',
   },
-  {
-    title: 'IT Journey',
-    detailsUrl: '/courses#learning-path',
-    description: 'Plan your developer path',
-  },
   {
     title: 'Mentoring',
-    detailsUrl: '/courses#mentors-wanted',
+    detailsUrl: '/#mentors-wanted',
     description: 'Contribute and study',
   },
 ];
 
+const communityMenuStaticLinks = [
+  {
+    title: 'About',
+    detailsUrl: '/community/#about',
+    description: 'Who we are',
+  },
+  {
+    title: 'Events',
+    detailsUrl: '/community/#events',
+    description: 'Meet us at events',
+  },
+  {
+    title: 'Merch',
+    detailsUrl: '/community/#merch',
+    description: 'Sloths for your daily life',
+  },
+  {
+    title: 'Contribute',
+    detailsUrl: '/community/#contribute',
+    description: 'Assist us and improve yourself',
+  },
+];
+
 interface SchoolMenuProps {
-  heading: 'rs school' | 'all courses';
+  heading: 'rs school' | 'all courses' | 'community';
+  hasTitle?: boolean;
   color?: 'dark' | 'light';
 }
 
-export const SchoolMenu = ({ heading, color = 'light' }: SchoolMenuProps) => {
+export const SchoolMenu = ({ heading, hasTitle = true, color = 'light' }: SchoolMenuProps) => {
   const { data } = useDataByName('courses');
 
   const courses = data as Course[];
 
-  const schoolListProps = heading.includes('courses') ? courses : schoolMenuStaticLinks;
+  const schoolListProps = heading.includes('courses')
```

Comment:
Let's do it through `if` and `else`

Reply:
Corrected in  [refactor: 256 - change the nested ternary operator to an if-else cons…](https://github.com/rolling-scopes/site/pull/258/commits/adf91c8ac67d3b9c0a01d0bd1bd6ac8804cc0b53)